packages/atlas/src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -432,7 +432,7 @@ export const CreateEditChannelView: FC<CreateEditChannelViewProps> = ({ newChann
               <Controller
                 name="title"
                 control={control}
-                rules={textFieldValidation({ name: 'Channel name', minLength: 3, maxLength: 40, required: true })}
+                rules={textFieldValidation({ name: 'Channel name', minLength: 3, maxLength: 40 })}
```

Comment:
Doesn't work correctly. Try to create a channel with 2 characters. You won't see errors and the button won't be disabled(it should - this is how it works on production at least)